ios/RNLinksdk.m

```diff
@@ -99,14 +99,16 @@ - (void)stopObserving {
     NSString *paymentTokenInput = [RCTConvert NSString:configuration[kRNLinkKitConfigPaymentTokenKey]];
     NSString *oauthStateId = [RCTConvert NSString:configuration[kRNLinkKitConfigOAuthStateIdKey]];
     NSString *institution = [RCTConvert NSString:configuration[kRNLinkKitConfigInstitutionKey]];
+    NSString *publicKey = [RCTConvert NSString:configuration[kRNLinkKitConfigInstitutionKey]];
 
     PLKConfiguration* linkConfiguration;
     BOOL isUsingLinkToken = [linkTokenInput length] > 0 && [linkTokenInput hasPrefix:kRNLinkKitLinkTokenPrefix];
+    BOOL isUsingPublicKey = kRNLinkKitConfigPublicKeyKey;
```

Comment:
this assignment is always going to be true. this should be checking if `publicKey` has length instead